### DISCUSSION TITLE
Service Response net_price_ct_per_kwh

### DISCUSSION THREAD

**Discussion by Joebinator** [mampfes/ha_epex_spot#63] [Q&A]
Can anyone explain to me what net prices means from the service response? 

If i ask the service for the lowest prices for 1 hour it gets me the price for that hour but when i ask for example 2 hours or longer i can not really make sense of the value.

I understand that the price_ct_per_kwh is added together but the net price seems not to, so i am trying to find out for what i can use this for.

**Comment by mampfes** [comment]
#60

Please note that the change is only available in main branch so far. I will create a release version soon.

**Comment by mampfes** [comment]
Fixed in release 2.2.0
